source/blog/2016-12-12-part2-install-origin-on-f25-atomic-host.html.md

```diff
@@ -0,0 +1,349 @@
+---
+title: 'Installing an OpenShift Origin Cluster on Fedora 25 Atomic Host: Part 2'
+author: dustymabe
+date: 2016-12-12 15:23:32 UTC
+tags: fedora, atomic, origin, openshift, kubernetes
+published: true
+comments: true
+---
+
+Introduction
+============
+
+In Part 1 of this series we used the [OpenShift Ansible
+Installer](https://github.com/openshift/openshift-ansible) to install
+[Openshift Origin](https://github.com/openshift/origin) on three servers
+that were running [Fedora 25 Atomic
+Host](https://getfedora.org/en/atomic/). From the [first
+post](http://www.projectatomic.io/blog/2016/12/part1-install-origin-on-f25-atomic-host/)
+the three machines we'll be using have the following roles and IP
+address configurations:
+
+```
+|    Role     |  Public IPv4   | Private IPv4 |
+|-------------|----------------|--------------|
+| master,etcd | 54.175.0.44    | 10.0.173.101 |
+| worker      | 52.91.115.81   | 10.0.156.20  |
+| worker      | 54.204.208.138 | 10.0.251.101 |
+```
+
+In this blog we'll explore the installed Origin cluster and then launch
+an application to see if everything works.
+
+READMORE                                                                                                                                                                                                         
+
+
+The Installed Origin Cluster
+============================
+
+With the cluster up an running we can log in as `admin` to the master
+node via the `oc` command. To install `oc` CLI on a your machine you can
+[follow these
+instructions](https://docs.openshift.org/1.2/cli_reference/get_started_cli.html#installing-the-cli)
```

Comment:
why link to 1.2 and not latest? https://docs.openshift.org/latest/cli_reference/get_started_cli.html#installing-the-cli